quantized_mesh_tile/utils.py

```diff
@@ -172,10 +169,9 @@ def computeNormals(vertices, faces):
 
     for i in xrange(0, numFaces):
         face = faces[i]
+        weightedNormal = [c * areasPerFace[i] for c in normalsPerFace[i]]
         for j in face:
-            weightedNormal = [c * areasPerFace[i] for c in normalsPerFace[i]]
-            normalsPerVertex[j] = c3d.add(
-                normalsPerVertex[j], weightedNormal)
+            normalsPerVertex[j] = c3d.add(normalsPerVertex[j], weightedNormal)
```

Comment:
In general yes, but this part of the code is not quite ready yet, that's why I put it in alpha and missed the octDecode stuff.